README.dev.md

```diff
@@ -204,6 +204,35 @@ host    all             all             ::1/128                 md5
 
 Now continue with the Common to Fedora/CentOS instructions below.
 
+### On OpenBSD 5.8 (experimental):
+
+These instructions are experimental and may have bugs; patches welcome!
+
+```
+doas pkg_add sudo bash gcc postgresql-server redis rabbitmq memcached node libmemcached py-Pillow py-cryptography py-cffi
+
+# Get tsearch_extras and build it (using a modified version which aliases int4 on OpenBSD):
+git clone https://github.com/blablacio/tsearch_extras
+cd tsearch_extras
+gmake && sudo gmake install
+
+# Point environment to custom include locations and use newer GCC (needed for Node modules):
+export CFLAGS="-I/usr/local/include -I/usr/local/include/sasl"
+export CXX=eg++
+
+# Create tsearch_data directory:
+sudo mkdir /var/postgresql/tsearch_data
+
+
+# Hack around missing dictionary files -- need to fix this to get
+# the proper dictionaries from what in debian is the hunspell-en-us package.
+sudo touch /usr/local/share/postgresql/tsearch_data/english.stop
+sudo touch /usr/local/share/postgresql/tsearch_data/en_us.dict
+sudo touch /usr/local/share/postgresql/tsearch_data/en_us.affix
```

Comment:
Should these be /var/postgresql/tsearch_data ?  Looking at the below that seems to be where the tsearch_data is going.